src/docs/asciidoc/getting-started.adoc

```diff
@@ -172,4 +172,5 @@ You have created a new application, executed it locally and then published it.
 Now, you are ready to code your own application:
 
 - See <<developer-manual.adoc#_develop_your_front_with_zetapush, how to use built-in ZetaPush cloud services>>
```

Comment:
The links will be updated. We were using automatic anchors based on titles but every time we rename a section, all links pointing to it are no more valid. 

Maybe we should use only manual anchors even if it takes more time. Otherwise, we have to check that every link is still ok...